iothub/service/src/DigitalTwin/Models/DigitalTwinGetResponse.cs

```diff
@@ -14,9 +14,12 @@ public class DigitalTwinGetResponse<T>
         /// <summary>
         /// Creates an instance of this class.
         /// </summary>
+        /// <remarks>
+        /// This class can be inherited from and set by unit tests for mocking purposes.
+        /// </remarks>
         /// <param name="eTag">Weak ETag of the modified resource.</param>
         /// <param name="digitalTwin">The deserialized digital twin.</param>
-        internal DigitalTwinGetResponse(T digitalTwin, ETag eTag = default)
+        protected internal DigitalTwinGetResponse(T digitalTwin, ETag eTag = default)
```

Comment:
A step further would be to make the properties that are initialized via a ctor as readonly (meaning no setter at all).

Reply:
Like in `DigitalTwinUpdateResponse`